vowpalwabbit/vw.sln

```diff
@@ -47,30 +41,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_testhelp", "..\cs\testhe
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "vw_common", "..\cs\common\vw_common.csproj", "{E621E022-C1F8-433F-905A-AB9A3DE072B7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_leaktest", "..\cs\leaktest\cs_leaktest.csproj", "{492EA3A7-8A41-459E-BBC3-5A3FE9DDFC68}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_testcommon", "..\cs\testcommon\cs_testcommon.csproj", "{6A9CBEAB-427F-4D8D-9559-B76B42B0895C}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "cs_vld", "..\cs\cli\cs_vld.vcxproj", "{3D57A6AF-DE8C-40DC-ABDE-E4CE1B9C0D20}"
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "vw_dynamic", "vw_dynamic.vcxproj", "{1E205806-7F80-47DD-A38D-FC08083F3593}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "vw_core", "vw_core.vcxproj", "{1E205806-7F80-47DD-A38D-FC08083F3593}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_console", "..\cs\cs_console\cs_console.csproj", "{01A85382-C3E9-480A-86BF-FAFE4AD107A7}"
 EndProject
-Project("{930C7802-8A8C-48F9-8165-68863BCCD9DD}") = "setup", "..\cs\setup\setup.wixproj", "{1F25D39D-71C3-4FDE-A58F-D3D5D4F6DFFB}"
-EndProject
-Project("{930C7802-8A8C-48F9-8165-68863BCCD9DD}") = "setup_bundle", "..\cs\setup_bundle\setup_bundle.wixproj", "{A80650FD-7DCA-4B80-B52E-D22D5CAE9E92}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_json", "..\cs\cs_json\cs_json.csproj", "{9E27FA94-AB34-4736-8427-FB7A2BA90D52}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_parallel", "..\cs\cs_parallel\cs_parallel.csproj", "{08636F79-5577-4AF2-8EED-EC8A5BC14AC4}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "azure", "..\cs\azure\azure.csproj", "{43E32C1D-21D6-4BE3-8FA5-D5BA379CBA53}"
-EndProject
-Project("{CC5FD16D-436D-48AD-A40C-5A424C6E3E79}") = "azure_service", "..\cs\azure_service\azure_service.ccproj", "{7263F536-3ABE-4A8E-BA66-1341D6A53579}"

```

Comment:
Any reason to keep the files for this project?

Reply:
Yes: I'd like to figure out whether we want to do something with these longer-term I just do not want them in the build until we decide.

Reply:
Got it.  Thanks